### PR TITLE
APPDEV-1198 Display user pic and smiley(for GoalConflictMessage) in notification list

### PR DESCRIPTION
**Pull request by vishal-revdiwala**
- Show adult-sad image if message type: GoalConflictMessage.
- Display user photo if added by user.
- Code refactored

### CODE REVIEW

File: Yona/Yona/CustomCells/YonaUserTableViewCell.swift [hunk 1]
```diff
@@ -116,18 +104,26 @@ class YonaUserTableViewCell: PKSwipeTableViewCell {
...
         }
+        
         normalLineLabel.text = "\(aMessage.nickname)"
      
         var tmpnickname = ""
         tmpnickname = aMessage.nickname
-        if tmpnickname.characters.count > 0 {
-            avatarNameLabel.text = "\(tmpnickname.capitalized.characters.first!)"
+        
+        if tmpnickname.count > 0 && aMessage.messageType != notificationType.GoalConflictMessage && aMessage.userPhotoLink == nil {
```
Comment (Bert-R) [inline Yona/Yona/CustomCells/YonaUserTableViewCell.swift]: If ``aMessage.messageType != notificationType.GoalConflictMessage``, then ``aMessage.userPhotoLink == nil`` is never true, so you can simplify this condition:
```
if tmpnickname.count > 0 && aMessage.userPhotoLink == nil {
```

File: Yona/Yona/CustomCells/YonaUserTableViewCell.swift [hunk 2]
```diff
@@ -116,18 +104,26 @@ class YonaUserTableViewCell: PKSwipeTableViewCell {
...
+                avatarImageView.kf.setImage(with: URL)
+            } else {
+                avatarImageView.image = nil
+            }
         }
+        
         normalLineLabel.text = "\(aMessage.nickname)"
      
         var tmpnickname = ""
         tmpnickname = aMessage.nickname
```
Comment (Bert-R) [inline Yona/Yona/CustomCells/YonaUserTableViewCell.swift]: Can't you remove ``tmpnickname`` and test on ``aMessage.nickname`` directly?.

File: Yona/Yona/CustomCells/YonaUserTableViewCell.swift (hunk 1 above)
Comment (vishal-revdiwala) [reply Yona/Yona/CustomCells/YonaUserTableViewCell.swift]: True. simplified condition.

File: Yona/Yona/CustomCells/YonaUserTableViewCell.swift (hunk 2 above)
Comment (vishal-revdiwala) [reply Yona/Yona/CustomCells/YonaUserTableViewCell.swift]: Yes, I can remove and compare directly. Updated code accordingly.

**Pull request merged**